pkgs/applications/networking/mullvad-vpn/default.nix

```diff
@@ -138,7 +138,8 @@ stdenv.mkDerivation {
     changelog = "https://github.com/mullvad/mullvadvpn-app/blob/${version}/CHANGELOG.md";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.gpl3Only;
-    platforms = [ "x86_64-linux" "aarch64-linux" ];
+    platforms = platforms.unix ;
+    badPlatforms = platforms.darwin;
```

Comment:
I think that this should be `badPlatforms = [ lib.systems.inspect.patterns.isDarwin ];` but I am not sure what is the difference.
cc @RossComputerGuy

Reply:
I'm not sure either, someone would have to run an eval but the patterns way is the more correct way I think.

Reply:
thanks for the correction!